### PR TITLE
Hypertable : fix infinite loading for text search

### PR DESCRIPTION
**Pull request by JulienVannier66**
### What does this PR do?

Fix infinite loading when search in text filter

Related to : https://github.com/upfluence/backlog/issues/304

### What are the observable changes?
Peek 2021-03-02 18-07

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- ~[ ] Added/updated tests~
- ~[ ] Added/updated documentation~
- [x] Properly labeled

### Impacted components

- [X] CRM :
- [ ] Mailings :
- [ ] Workflow :
- [ ] Search :
- [ ] Side Panel :
- [ ] Analytics :
- [ ] Live Capture :
- [ ] Payments :
- [ ] Plugin :
- [ ] Stores :
  - [ ] PG store : 
  - [ ] ES store : 
  - [ ] Cassandra store : 
- [ ] Harvest : 
- [ ] Infrastructure : 
- [ ] Other : 
- [ ] No impact

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- [ ] Plugin :
+- [ ] Stores :
+  - [ ] PG store : <!-- Inbox, Analytics, Influencer ... -->
+  - [ ] ES store : <!-- Inbox, Analytics, Influencer ... -->
+  - [ ] Cassandra store : <!-- Activity, Graph-server ... -->
+- [ ] Harvest : <!-- Scrappers -->
+- [ ] Infrastructure : <!-- UDS, Vulcan ... -->
+- [ ] Other : <!-- Your changes concern something more particular -->
+- [ ] No impact <!-- Documentation for example -->
+
```
Comment (phndiaye) [inline .github/pull_request_template.md:51]: @jlevesy what about adapting for frontend projects ? :)

File: addon/components/hyper-table/filters-renderers/text.js [hunk 2]
```diff
@@ -48,6 +44,13 @@ export default Component.extend(FiltersRendererMixin, {
     run.debounce(this, this._addSearchFilter, 1000);
   }),
 
+  init() {
+    this._super();
+
+    let searchTerm = this.column.filters.findBy('key', this.searchKey || 'value');
+    this.set('_searchQuery', searchTerm?.value);
+  },
+
```
Comment (phndiaye) [inline addon/components/hyper-table/filters-renderers/text.js:53]: CLEAN 👍

**Review by phndiaye — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (jlevesy) [reply .github/pull_request_template.md:51]: Sure, go ahead ! What do you have in mind ?

File: addon/components/hyper-table/filters-renderers/text.js (hunk 2 above)
Comment (JulienVannier66) [reply addon/components/hyper-table/filters-renderers/text.js:53]: Thanks to @Hegazy360 for helping me

File: .github/pull_request_template.md (hunk 1 above)
Comment (JulienVannier66) [reply .github/pull_request_template.md:51]: For this project, I find it relevant. Especially if we want to use it in other projects

File: addon/components/hyper-table/filters-renderers/text.js [hunk 3]
```diff
@@ -48,6 +44,13 @@ export default Component.extend(FiltersRendererMixin, {
     run.debounce(this, this._addSearchFilter, 1000);
   }),
 
+  init() {
+    this._super();
+
+    let searchTerm = this.column.filters.findBy('key', this.searchKey || 'value');
```
Comment (jlevesy) [inline addon/components/hyper-table/filters-renderers/text.js:50]:
```suggestion
    const searchTerm = this.column.filters.findBy('key', this.searchKey || 'value');
```
:thinking: ?

Comment (JulienVannier66) [reply addon/components/hyper-table/filters-renderers/text.js:50]: I change that :)

File: .github/pull_request_template.md (hunk 1 above)
Comment (phndiaye) [reply .github/pull_request_template.md:51]: The only option relevant that could ever be ticked here is CRM. And things like `Stores`, `Harvest`, etc. are hardly reachable from the front 😅 

@jlevesy  For the general frontend case, I would've added the Developer Heads Up about Octane here. (it's just that the pull request template never had

Comment (JulienVannier66) [reply .github/pull_request_template.md:51]: @phndiaye @jlevesy  I can add Developer Heads Up about Octane here and change the template with "classic" front template. What do you think?

Comment (jlevesy) [reply .github/pull_request_template.md:51]: Yes go ahead.

**Review by jlevesy — APPROVED** [review]
LGTM (when you change the template)

**Pull request merged**